cli/internal/shell.go

```diff
@@ -0,0 +1,30 @@
+package internal
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Execute execute
```

Comment:
I'm not sure what this comment means. :)

Reply:
Let me complete it. My mistake 😂